CMakeLists.txt

```diff
@@ -129,7 +129,14 @@ IF(HAVE_EXECINFO_H)
       ADD_DEFINITIONS(-DHAVE_EXECINFO_H)
     ENDIF()
   ELSE()
-    ADD_DEFINITIONS(-DHAVE_EXECINFO_H)
+    IF(CMAKE_SYSTEM_NAME STREQUAL "NetBSD")
```

Comment:
Why not `CMake/FindBacktrace.cmake`?

Reply:
Probably because I'm clueless about cmake. I was just using the mechanisms I could see being used in CMakeLists.txt already. If this should have been done differently, please show me how!

Reply:
```
find_package(Backtrace)
```

You can use `${Backtrace_FOUND}`, `${Backtrace_HEADER}`.

Reply:
I really don't understand that, Kamil. The fragment of cmakefile you tie this to is part of what's a pretty ugly nest of conditionals - is that what you'd like to see cleaned up? I've discovered that cmake has elseif(), which I would have used for the NetBSD part if I'd known. It should have been used already for the FreeBSD bit, but seeing as it wasn't, I just assumed it had to be that way, and continued in the same fashion.

Reply:
Please detect Backtrace without homegrown OS-specific checks.

Reply:
I finally see what you mean. Sorry for being so slow. :)

I think changing the mechanism for resolving the execinfo/backtrace thing is beyond the scope of my just getting the existing implementation to work on NetBSD, and it would be overly ambitious for me to submit patches to do so.

What I will do is to simplify the conditional clauses a bit: I've added more complex nesting than I needed to in a couple of places, and I've also got that test for libusb1, which I can actually just lose, because I've since sorted that out in open-zwave itself, which is where it's needed.